components/Sections/NewHeader/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { FormattedMessage } from 'react-intl';
+import Image from 'next/image';
+import { MdSearch } from 'react-icons/md';
+import Link from 'next/link';
+import styles from './index.module.scss';
+import ActiveLocalizedLink from '../../ActiveLocalizedLink';
+import DarkModeToggle from '../../Common/DarkModeToggle';
+
+const Header = () => (
+  <nav aria-label="Primary" className={styles.header}>
+    <div className={styles.container}>
+      <div className={styles.startWrapper}>
+        <Link href="/" aria-label="Homepage">
+          <div className={styles.logo}>
+            <div>
+              <Image
+                src={'static/images/logo-light.svg'}
+                alt="light-logo"
+                className="light-mode-only"
+                width="111"
+                height="33"
+              />
+            </div>
+            <div>
+              <Image
+                src={'static/images/logo.svg'}
+                alt="dark-logo"
+                className="dark-mode-only"
+                width="111"
+                height="33"
+              />
+            </div>
+          </div>
+        </Link>
+      </div>
+
+      <ul className={styles.tabs}>
+        <li>
```

Comment:
"Certification" is missing.
I did not find a comment explaining the removing (an oversight?)

Reply:
And about "about" ?

Reply:
Both are missing from the figma design, so I also haven't added them here.

Reply:
Figma is good entry point but you have to take a look to nodejs.dev to see how it's render.

Reply:
Got ya